src/app/footer/footer.component.html

```diff
@@ -0,0 +1,68 @@
+                <div class="widget widget-links widget-light pb-2 mb-4">
+                    <h3 class="widget-title text-light">{{ shopDepartment }}</h3>
+                    <ul class="widget-list">
+                        <li class="widget-list-item" *ngFor= "let shopDepartment of shopDepartments; let i=index;" >
+                            <a class="widget-list-link" id={{i}} href="#">{{ shopDepartment.name }}</a>
+                        </li>     
+                    </ul>    
+                </div>
+            </div>
+            <div class="col-md-4 col-sm-6">
+                <div class="widget widget-links widget-light pb-2 mb-4">
+                    <h3 class="widget-title text-light">{{ shippingInfo }}</h3>
+                    <ul class="widget-list">
+                        <li class="widget-list-item" *ngFor= "let shipInfo of shippingTabs;let i=index;">
+                            <a class="widget-list-link" id={{i}} href="#">{{ shipInfo.name }}</a>
+                        </li> 
+                    </ul>  
+                </div>  
+                <div class="widget widget-links widget-light pb-2 mb-4">
+                    <h3 class="widget-title text-light">{{ aboutUs }}</h3>
+                    <ul class="widget-list">
+                        <li class="widget-list-item" *ngFor= "let about of aboutKart;let i=index;">
+                            <a class="widget-list-link"  id={{i}} href="#">{{ about.name }}</a>
+                        </li>   
+                    </ul>  
+                </div> 
+            </div>
+            <div class="col-md-4">
+                <div class="widget pb-2 mb-4">
+                    <h3 class="widget-title text-light pb-1">Download our app</h3>
+                    <div class="d-flex flex-wrap">
+                        <div class="me-2 mb-2">
+                            <a class="btn-market btn-apple" href="#" role="button">
+                                <span class="btn-market-subtitle">Download on the </span>
+                                <span class="btn-market-title">App Store</span>
+                            </a>
+                        </div>  
+                        <div class="mb-2">
+                            <a class="btn-market btn-google" href="#" role="button">
+                                <span class="btn-market-subtitle">Download on the </span>
+                                <span class="btn-market-title">Google Play</span>
+                            </a>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>    
+    </div>  
+</div>    
+<div class="pt-5 bg-footer-slighter">
+    <div class="container">
+        <div class="row pb-3">
+            <div class="col-md-3 col-sm-6 mb-4" *ngFor="let label of footerLabels;let i=index;">
+                <div class="d-flex">
+                    <i class= {{label.className}}  style="font-size: 2.40rem;"></i>
```

Comment:
do not use style in html. You can add class if you want.

Reply:
@akashsinha13, yes we are handling that property in CSS file.